src/frontends/paddle/tests/paddle_pip_check.py

```diff
@@ -1,20 +1,41 @@
 import pkg_resources
 import re
 import sys
+import tempfile
+from itertools import islice
 
 req_file=sys.argv[1]
+con_file=sys.argv[2]
+
+tmpfile = tempfile.TemporaryFile(mode="w+")
+
+confile=open(con_file, mode='r')
+for line in confile.readlines():
+    tmpfile.write(line)
+tmpfile.write("\n")
+reqfile=open(req_file, mode='r')
+for line in islice(reqfile, 2, None):
```

Comment:
can we parse confine from reqfile? without adding extra arguments

Reply:
done